prov/psm3/psm3/ptl_am/am_reqrep_shmem.c

```diff
@@ -1886,7 +1886,8 @@ void process_packet(ptl_t *ptl_gen, am_pkt_short_t *pkt, int isreq)
 		switch (pkt->type) {
 		case AMFMT_LONG_END:
 			isend = 1;
-		case AMFMT_LONG:
+		/* fall through */
+		case AMFMT_LONG: /* fall through */
```

Comment:
nit: you don't need fall through for back to back case statements.

Reply:
Fixed in #6572